engine/src/main/resources/application.conf

```diff
@@ -218,27 +218,13 @@ database {
   }
 
   test {
-    hsqldb {
-      db.url = "jdbc:hsqldb:mem:testdb;shutdown=false;hsqldb.tx=mvcc"
-      db.driver = "org.hsqldb.jdbcDriver"
-      db.connectionTimeout = 1000 // NOTE: 1000ms (the default) is ok for a small hsqldb, but often too short for mysql
-      driver = "slick.driver.HsqldbDriver$"
-      liquibase = {
-        changelog = "src/main/migrations/changelog.xml"
-        connection = "liquibase.database.jvm.HsqlConnection"
-      }
-    }
-
```

Comment:
It was used to test liquibase-on-hsqldb when mysql wasn't available.

Now that we use liquibase-on-hsqldb for all schema creation, test.hsqldb is an exact duplicate of the main.hsqldb tests.